archetypes/database-se/src/main/resources/src/main/java/__pkg__/PokemonService.java.mustache

```diff
@@ -52,25 +52,17 @@ public class PokemonService implements Service {
     }
 
     private void listTypes(ServerRequest request, ServerResponse response) {
-        try {
-            List<PokemonType> pokemonTypes =
-                    dbClient.execute(exec -> exec.namedQuery("select-all-types"))
-                            .map(row -> row.as(PokemonType.class)).collectList().get();
-            response.send(pokemonTypes);
-        } catch (InterruptedException | ExecutionException e) {
-            throw new RuntimeException(e);
-        }
+        dbClient.execute(exec -> exec.namedQuery("select-all-types"))
```

Comment:
This approach will collect all records into memory.
We can use:
```
response.send(dbClient.execute(exec -> exec.namedQuery("select-all-types"))
                .map(row -> row.as(PokemonType.class)), PokemonType.class);
```
To use JSON-Binding streaming support in Helidon to send records using publisher directly.